problem-spring-webflux/src/test/java/org/zalando/problem/spring/webflux/advice/validation/BindAdviceTraitTest.java

```diff
@@ -12,8 +13,15 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.startsWith;
 
+import java.util.Locale;
+
 final class BindAdviceTraitTest implements AdviceTraitTesting {
 
+    @BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/problem-spring-web/pullRequest?prid=4540504)